manager/integration/tests/test_cloning.py

```diff
@@ -392,7 +393,10 @@ def test_cloning_interrupted(client, core_api, pvc, pod, clone_pvc, clone_pod):
     crash_replica_processes(client, core_api, source_volume_name)
 
     # Step-7
-    wait_for_volume_faulted(client, source_volume_name)
+    wait_for_volume_status(client, source_volume_name,
```

Comment:
Could you check other cases also flaky as well when using `crash_replica_processes` + `wait_for_volume_faulted` together?

I think there are other places using these as well.

Reply:
Yes in `test_ha.py`, test case `test_ha_salvage` was [flaky](https://ci.longhorn.io/job/public/job/master/job/sles/job/amd64/job/longhorn-tests-sles-amd64/209/testReport/tests/test_ha/test_ha_salvage/) due to waiting volume faulted status.
I will submit another PR for the flaky case